x-pack/plugin/relevance-search/src/main/java/org/elasticsearch/xpack/relevancesearch/relevance/boosts/ScriptScoreBoost.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.relevancesearch.relevance.boosts;
+
+import java.text.MessageFormat;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Objects;
+
+public abstract class ScriptScoreBoost {
+    protected String type;
+    protected String operation = "multiply";
```

Comment:
Should `operation` be an enum or at least should valid operations be constants?

Reply:
I agree. It's one of those "quick and dirty code warnings". I'll clean up some of them.